src/nomics_tests.py

```diff
@@ -0,0 +1,39 @@
+from dextrader.nomics.utils import get_candles, format_query_as_dataframe, get_recent_trades
+from APIKEYS import API_KEY
+import os
+
+
```

Comment:
Thanks for adding the tests, as this also illustrates usage.

Reply:
np!